webhook-connector/order-events.md

```diff
@@ -0,0 +1,131 @@
+| ------------------------------- | ----------------------------------------- |
+| `OrderLinesAcceptedBySupplier`  | Order lines are accepted by the supplier  |
+| `OrderLinesRejectedBySupplier`  | Order lines are rejected by the supplier  |
+| `OrderLinesConfirmedBySupplier` | Order lines are confirmed by the supplier |
+| `OrderLinesConfirmedByBuyer`    | Order lines are confirmed by the buyer    |
+
+
+## Order proposal by supplier
+
+The supplier can propose an alternative delivery schedule or prices.
+
+The buyer either approves or rejects the proposal.
+
+| OrderEvent                            | Webhook Configuration                                        |
+| --------------------------------------| ------------------------------------------------------------ |
+| `OrderChangesProposedBySupplier`      | Order changes are proposed by the supplier                   |
+| `OrderChangesProposalApprovedByBuyer` | By supplier proposed order changes are approved by the buyer |
+| `OrderChangesProposalRejectedByBuyer` | By supplier proposed order changes are rejected by the buyer |
+
+## Order updated by supplier
+
+The supplier can update existing order lines.
+
+If delivery schedule or prices are updated this will become either a proposal or reopen request.
+
+| OrderEvent                     | Webhook Configuration                        |
+| ------------------------------ | -------------------------------------------- |
+| `OrderLinesUpdatedBySupplier`  | Order and line fields other than prices & delivery schedules are updated by the supplier |
+| `OrderLinesItemDetailsChanged` | Order lines item details are changed by the supplier | 
+
+## Order reopen request by buyer
+
+The buyer can request to reopen confirmed order lines.
+
+The supplier can either approve or reject the reopen request.
+
+| OrderEvent                                  | Webhook Configuration                        |
+| ------------------------------------------- | -------------------------------------------- |
+| `OrderLinesReopenRequestedByBuyer`          | Order lines reopen is requested by buyer     |
+| `OrderLinesReopenRequestApprovedBySupplier` | Buyer reopen request is approved by supplier |
+| `OrderLinesReopenRequestRejectedBySupplier` | Buyer reopen request is rejected by supplier |
+
+## Order reopen request by supplier
+
+The supplier can request to reopen confirmed order lines.
+
+The buyer can either approve or reject the reopen request.
+
+| OrderEvent                               | Webhook Configuration                        |
+| ---------------------------------------- | -------------------------------------------- |
+| `OrderLinesReopenRequestedBySupplier`    | Order lines reopen is requested by supplier  |
+| `OrderLinesReopenRequestApprovedByBuyer` | Supplier reopen request is approved by buyer |
+| `OrderLinesReopenRequestRejectedByBuyer` | Supplier reopen request is rejected by buyer |
```

Comment:
```suggestion
| `OrderLinesReopenRequestedBySupplier`    | Order lines reopen is requested by the supplier  |
| `OrderLinesReopenRequestApprovedByBuyer` | Supplier reopen request is approved by the buyer |
| `OrderLinesReopenRequestRejectedByBuyer` | Supplier reopen request is rejected by the buyer |
```

Reply:
The text in the table are the literal texts in the webhook configuration you may choose.

Reply:
I will commit anyway, portal to be improved